### PR TITLE
Rename 1990-rjsf to 1990

### PR DESCRIPTION
**Pull request by jbalboni**
This moves everything from 1990-rjsf to 1990 and updates all code references.

### CODE REVIEW

File: src/sass/edu-benefits.scss [hunk 1]
```diff
@@ -9,141 +9,14 @@
 @import "modules/m-omb-info";
 @import "modules/m-form-confirmation";
 
-// 1990
```
Comment (cvalarida) [inline src/sass/edu-benefits.scss]: Woo! Delete all the things!

File: test/e2e/edu-1990-rjsf-helpers.js [hunk 2]
```diff
@@ -1,90 +0,0 @@
-import _ from 'lodash/fp';
```
Comment (cvalarida) [inline test/e2e/edu-1990-rjsf-helpers.js]: Looks like this was all just copied into `edu-1990-helpers.js`, aye? This diff isn't super helpful. 😒

**Review by cvalarida — APPROVED** [review]

File: test/e2e/edu-1990-rjsf-helpers.js (hunk 2 above)
Comment (jbalboni) [reply test/e2e/edu-1990-rjsf-helpers.js]: Yeah, it's not. I just renamed this file.

**Review by cehsu — APPROVED** [review]
LGTM.

**Pull request merged**